Python/UpeachyaKansakar/tasks/task2/scraping.py

```diff
@@ -0,0 +1,64 @@
+import time
+import requests
+import urllib.request
+from pandas import DataFrame 
+from bs4 import BeautifulSoup
+
+URL = 'https://www.opencodez.com/category/web-development'
+
+def scrap_page_title(soupObject):
```

Comment:
```suggestion
def scrap_page_title(soup_object):
```
Please follow snake case while naming all the variable name

Reply:
:ok: